snuba/datasets/discover.py

```diff
@@ -74,99 +71,31 @@ def detect_dataset(
     return EVENTS
 
 
-class DiscoverSource(RelationalSource):
```

Comment:
The responsibility of this is now done in a generic way by the QuryPlanBuilder through DiscoverQueryStorageSelector